projects/angular/src/button/_buttons.clarity.scss

```diff
@@ -5,151 +5,99 @@
  */
 @use 'sass:meta';
 @use '../utils/mixins';
-@use '../utils/variables/variables';
-@use 'variables.buttons' as buttons-variables;
```

Comment:
I don't understand. All CSS vars (eg `--clr-btn-appearance-standard-height`) are tied to CDS tokens (eg `#{tokens.$cds-global-space-9}`). Old CLR variables are not used any more and have no references in the files.

Is your suggestion to tie CLR varables to tokens (eg: `$clr-btn-appearance-standard-height: {tokens.$cds-global-space-9} !default;`) and then reference them as fallbacks?

Reply:
The problem is developers can still misstype `--clr-btn` properties. In the same way that core provides the token SCSS variables that map to CSS vars in the tokens file we're importing, we should follow the same pattern. This would keep the properties file, where we should map SCSS variables to the appropriate --clr or --cds token, and this file would only use the scss variable. I'm not asking you to do it for all of the variables, but that file should be kept and CSS vars that we're using in this file should be mapped via a SCSS variable from the properties file. 

CDE-864 is the related ticket to provide SCSS variables for all --clr variables in order to get compile time checking.

Reply:
If I understood correctly the flow will be to reference CLR variable that is mapped to CLR property which is mapped to CDS variable which i mapped to CDS property.

Example
`
$clr-btn-appearance-standard-height: var(--clr-btn-appearance-standard-height);` -> `--clr-btn-appearance-standard-height: {tokens.$cds-global-space-9};` -> ` $cds-global-space-9: var(--cds-global-space-9)` -> `var(--cds-global-space-9): 16px`.
And in buttons `.btn { height: button-vars.$clr-btn-appearance-standard-height; }`

Reply:
Yes. That ensures that every instance of us using the $clr-btn- scss variable will be checked at compile time. We only need to thoroughly validate it is correctly spelled in one place despite potentially using it in multiple places. It also gets us closer to our secondary goal of exporting a file of SCSS variables so our product teams can get compile-time checking as well.

Reply:
We are following 2 approaches.

1.  One is consistent with static variable property assignment ` $clr-btn-appearance-standard-height: var(--clr-btn-appearance-standard-height);` . With `if {} else {}` strategy. [Line 54 through 117](https://github.com/vmware-clarity/ng-clarity/blob/buttons-remove-shim_cde-1214/projects/angular/src/button/_variables.buttons.scss#L54).
2. The other is using map `$clr-btn-primary: ( color: var(--clr-btn-primary-color), ... ) !default;`. With `map-get` strategy.[line 252 through 269](https://github.com/vmware-clarity/ng-clarity/blob/buttons-remove-shim_cde-1214/projects/angular/src/button/_variables.buttons.scss#L252).

Question is for such a variable bundle which will be the better option. The second defiantly will NOT follow the requirement.

Reply:
We can address the second strategy later as part of CDE-864.